gtfu/src/main/java/gtfu/tools/UpdateTripNames.java

```diff
@@ -0,0 +1,121 @@
+
+            String filePath = "server/agency-config/gtfs/gtfs-aux/" + agencyID +"/trip-names.json";
+            long lastUpdatedTripList = gh.getLatestCommitMillis(filePath);
+            Debug.log("- lastUpdatedTripList: " + lastUpdatedTripList);
+
+            String gtfsURL = yml.getURL(agencyID);
+            long lastModifiedLRemote = Util.getLastModifiedRemote(gtfsURL);
+            Debug.log("- lastModifiedLRemote: " + lastModifiedLRemote);
+
+            if(lastModifiedLRemote > lastUpdatedTripList){
+                Debug.log("Static GTFS has been updated since trip-names.json was last updated. Re-running TripListGenerator now to check whether this impacts trip-names.json");
+
+                String fileURL = "https://raw.githubusercontent.com/cal-itp/graas/main/" + filePath;
+                ProgressObserver po = new ConsoleProgressObserver(40);
+                byte[] currentFile = Util.getURLContentBytes(fileURL, po);
+
+                ByteArrayOutputStream baos = new ByteArrayOutputStream();
+                String utf8 = StandardCharsets.UTF_8.name();
+                try (PrintStream ps = new PrintStream(baos, true, utf8)) {
+                    TripListGenerator.generateTripList(agencyID, null, ps);
+                }
+                byte[] newFile = baos.toByteArray();
+
+                if(!Arrays.equals(currentFile, newFile)){
+                    Debug.log("Relevant changes detected. Creating a PR to update the file");
+
+                    // Consider linking the URL for the PR. This requires some updates to Sendgrid.java, to send HTML.
+                    reporter.addLine(agencyID);
+
+                    String title = ":robot: updates to " + agencyID + " triplist";
+                    String description = "Our automated daily check detected that changes were made to " + agencyID + "'s static GTFS. This PR was automatically generated, so please review and make updates if necessary before merging";
+                    String message = "Update trip-names.json to reflect static GTFS updates";
+                    String branchName = agencyID + "-triplist-update-" + Util.now();
+                    gh.createCommitAndPR(title, description, filePath, newFile, message, branchName);
+                    prCount++;
+                }
+                else{
+                    Debug.log("No relevant changes detected.");
+                }
+            }
+            else{
+                Debug.log("trip-names.json has been updated since the last static GTFS update. Continuing.");
+            }
+        }
+        if (prCount == 0) {
+            reporter.addLine("...no agencies. Everything looks up to date.");
+        }
+        reporter.send();
+    }
+
+    private static void usage() {
+        System.err.println("usage: UpdateTripNames [-u|--url <live-agencies-url>] [-a|--agency-id <agency-id>]");
+        System.err.println("    <url> is assumed to point to a plain text document that has an agency ID per line");
```

Comment:
<url> is not the name used two lines up. Please also add blurb for agency-id, and mention that either -u or -a must be given.

Reply:
Updated the blurb but not sure what you mean by "not the name used 2 lines up"